examples/README.md

```diff
@@ -3,8 +3,9 @@
 This folder contains examples of using the helm chart with various configurations and features enabled. Check each example's README file for more details:
 
 - [Custom certificate issuer](./with-custom-cert-issuer/README.md)
-- [External access](./with-external-access/README.md)
-- [Load balancing](./with-load-balancing/README.md)
+- [External access via NodePort](./with-external-via-nodeport/README.md)
+- [External access via LoadBalancer](./with-external-via-loadbalancer/README.md)
+- [External access via Ingress](./with-external-via-ingress/README.md)
```

Comment:
These files are what will be added in later PRs (one per file). Right now they don't exist.